config.json

```
@@ -462,6 +462,18 @@
           "functions"
         ]
       },
+      {
+        "slug": "high-scores",
+        "name": "High Scores",
+        "uuid": "2c81f99a-991a-4914-aae7-d5bf72456119",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 2,
```

```suggestion
        "difficulty": 3,
```

I think I'd bump this up to at least 3 given the use of output parameters.

---

Done